src/snowflake/snowpark/modin/plugin/docstrings/base.py

```diff
@@ -1603,8 +1592,6 @@ def loc():
         -----
         To meet the nature of lazy evaluation:
 
-        - Snowpark pandas ``.loc`` ignores out-of-bounds indexing for all types of indexers (while pandas ``.loc``
```

Comment:
We still ignores out-of-bounds indexing on the index, e.g., series.loc['nonexist']. Can you clarify it?

Reply:
I added a clarification bullet below this and changed the doc here.